apps/box/src/constants/layout.ts

```diff
@@ -0,0 +1,38 @@
+import { Dimensions } from 'react-native';
+
+export const SCREEN_WIDTH = Dimensions.get('window').width;
+export const SCREEN_HEIGHT = Dimensions.get('window').height;
+
+// Guideline sizes are based on figma
+const baseWidth = 375;
+const baseHeight = 812;
+
+/**
+ * Scale any size based off screen width with a moderating factor.
+ * sizes will scale in the ratio of (SCREEN_WIDTH / baseWidth) at a rate of some factor.
+ * If no factor is provided this will scale with a factor of 1
+ * @param size base size you wish to scale.
+ * @param factor rate at which to scale up.
+ */
+export const scaleByWidth = (size = 0, factor = 1) =>
```

Comment:
Where do we plan on using these?

Reply:
For the images on the mode screen, so the scale for bigger and smaller phones.